reana_client/utils.py

```diff
@@ -209,3 +209,25 @@ def get_analysis_root():
                 analysis_root = parent_dir
     analysis_root += '/'
     return analysis_root
+
+
+def cli_printer(headers, _filter, data):
```

Comment:
We could call this `cli_table_printer()` to be more descriptive, but we can refactor when we move this to `reana-commons`...